src/functions/users.rs

```diff
@@ -66,6 +66,21 @@ pub fn new_user(username: &str, hasroot: bool, password: &str, do_hash_pass: boo
             files::append_file("/mnt/etc/sudoers", "\nDefaults pwfeedback\n"),
             "Add pwfeedback to sudoers",
         );
+        files_eval(
+            Ok(files::create_file(&format!(
```

Comment:
Why is the `files::create_file` wrapped in a `Ok()`?

Reply:
`create_file` does not return a `Result<>` like other functions. Not sure if this is intended or not

Reply:
`create_file` does it's own error handling, it's not supposed to be wrapped in `files_eval`

Reply:
Alright, I'll edit this accordingly